kedro_mlflow/template/project/mlflow.yml

```diff
@@ -25,6 +25,13 @@ run:
   name: null # if `name` is None, pipeline name will be used for the run name
   nested: True  # # if `nested` is False, you won't be able to launch sub-runs inside your nodes
 
+hooks:
+  node:
+    flatten_dict_params: False  # if True, parameter which are dictionary will be splitted in multiple parameters when logged in mlflow, one for each key.
+    recursive: True  # Should the dictionnary flattening be applied recursively (i.e for nested dictionaries)? Not use if `flatten_dict_params` is False.
```

Comment:
Typo: dictionnary -> dictionary

Reply:
Indeed. This typo is very common across the project (double "n" is the french spelling), I've fixed it everywhere on the way.